.github/workflows/pkg-platform.yml

```diff
@@ -218,7 +218,7 @@ jobs:
         if: ${{ ! inputs.dry-run }}
 
   complain:
-    needs: [build, test, bottle]
+    needs: bottle
```

Comment:
this was pointless readability optimization that I hope doesn’t cause it to fail to complain on build failure.